reference/reference_test.go

```diff
@@ -152,6 +169,12 @@ func TestReferenceParse(t *testing.T) {
 		},
 	}
 	for _, testcase := range referenceTestcases {
+		if testcase.input == "Uppercase/lowercase:tag" {
+			// FIXME "Uppercase" is incorrectly handled as a domain-name here, therefore passes.
+			// See https://github.com/docker/distribution/pull/1778, and https://github.com/docker/docker/pull/20175
+			t.Skip("skipping 'Uppercase/lowercase:tag'")
```

Comment:
oh, darn, yes could've thought of that myself. I can update, but may take slightly longer (vacation)

> On 22 Aug 2016, at 02:45, Aaron Lehmann <notifications@github.com> wrote:
> 
> In reference/reference_test.go:
> 
> > @@ -152,6 +169,12 @@ func TestReferenceParse(t *testing.T) {
> >  		},
> >  	}
> >  	for _, testcase := range referenceTestcases {
> > +		if testcase.input == "Uppercase/lowercase:tag" {
> > +			// FIXME "Uppercase" is incorrectly handled as a domain-name here, therefore passes.
> > +			// See https://github.com/docker/distribution/pull/1778, and https://github.com/docker/docker/pull/20175
> > +			t.Skip("skipping 'Uppercase/lowercase:tag'")
> Skip will stop execution of the test; probably not what you want here. Sorry for suggesting it - I misunderstood and thought the whole test was problematic.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>